test/api/core/hotelClerk/addSubscription.test.js

```diff
@@ -206,7 +206,6 @@ describe('Test: hotelClerk.addSubscription', () => {
       id: 'foobar',
     }));
     return hotelClerk.addSubscription(request)
-      .then(() => {throw new Error('should not happen');})
```

Comment:
This should not be removed: if `hotelClerk.addSubscription` resolves instead of rejecting with an error, the test incorrectly passes.

Reply:
Hmm, its a mistake, i cant see why i did that